tests/test_hash.c

```diff
@@ -21,13 +21,14 @@
 #define TESTSIZE 500
 
 /* static test data */
-const int nkeys = 5;
+#include "test.h"
 const char *keys[] = {
   "foo", "bar", "baz", "quux", "xyzzy"
 };
 const char *values[] = {
   "wuzzle", "mug", "canonical", "rosebud", "lottery"
 };
+const int nkeys = ARRAY_SIZE(keys); /* move this line below the `values` array */
```

Comment:
Please, remove the comment. It was just a hint how to apply my example in the code. Sorry for confusion.